lsp-ada.el

```diff
@@ -0,0 +1,73 @@
+;;; lsp-ada.el --- description -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 E. Alexander Barbosa
```

Comment:
I still think this should be [lsp maintainers](https://github.com/emacs-lsp/lsp-mode/blob/master/lsp-headerline.el#L3) since everybody will maintain this

Reply:
yep, I forgot to ask about that :)

Reply:
done